includes/theme.php

```diff
@@ -289,7 +289,9 @@ function ap_post_actions( $_post = null ) {
 		$actions[] = array( 'header' => true );
 	}
 
-	if ( ap_user_can_delete_post( $_post->ID ) ) {
+	$answers = ap_count_published_answers( $_post->ID );
+
+	if ( ap_user_can_delete_post( $_post->ID ) && ( ! ap_opt( 'trashing_question_with_answer' ) || ( ap_opt( 'trashing_question_with_answer' ) && empty( $answers ) ) ) ) {
```

Comment:
@mebishalnapit Can this be simplified to this:
```
if (ap_user_can_delete_post($_post->ID) && (!ap_opt('trashing_question_with_answer') || empty($answers))) {
```

Reply:
@rahularyan Modified the condition as requested.